### PR TITLE
[cmsv2] CMS redesign work

### PR DESCRIPTION
**Pull request by amassarwi**
This diff adds all CMS redesign work.
I've done the following in order to sync the branches:
 1. Merged master into cmsv2 to resolve all conflicts
 2. Updates react-script to fix build issue
 3. I've deleted `react-select` dep - it's now part of `pi-ui`
 4. Finally deleted  renamed `componentsv2` to `components` & removed old folder content

**Resolved - see update in comment below**
~~For some reason build is failing currently with the following error:~~
```
Failed to compile.

./node_modules/react-dev-utils/webpackHotDevClient.js
Error: [BABEL] /home/amass/src/politeiagui/node_modules/react-dev-utils/webpackHotDevClient.js: Could not find plugin "proposal-numeric-separator". Ensure there is an entry in ./available-plugins.js for it. (While processing: "/home/amass/src/politeiagui/node_modules/babel-preset-react-app/dependencies.js$0")
    at Array.map (<anonymous>)
    at Generator.next (<anonymous>)
    at Generator.next (<anonymous>)
```

~~**Update:** It seems like the build error is related to: https://github.com/facebook/create-react-app/issues/8680 - I've tried the suggested workaround here https://github.com/facebook/create-react-app/issues/8680#issuecomment-601851525 but it didn't resolve the issue.~~

### CODE REVIEW

File: src/reducers/api/handlersCMS.js [hunk 1]
```diff
@@ -223,3 +223,17 @@ export const onReceiveCensorDCCComment = (state, action) => {
     }
   };
 };
+
+export const onRequestLikeComment = (state, action) => {
```
Comment (amassarwi) [inline src/reducers/api/handlersCMS.js]: Originally was part of `handlers.js` file on master - which does not exist anymore on cmsv2.
Not sure if `./handlersCMS.js` is the right place for it tho

**Comment by fernandoabolafio** [comment]
I love this `+7,977 −24,019`!

Ok, so I found a couple of things when running the Proposals app. I don't know exactly if they were all generated by the merging or not but I think it is worth investigating anyway. We may want to either cover it on this PR or create new issues after the merging. 

- User-submitted proposals always showing a loading state:
Screenshot 2020-03-22 at 12 54 08

- When 'setting an user as paid', I click on confirm and it only clears the input instead of performing the action:
Mar-22-2020 13-31-53

- Misplaced icons on the bottom of the proposal detail card:
Screenshot 2020-03-22 at 13 26 31

File: src/reducers/api/handlersCMS.js (hunk 1 above)
Comment (tiagoalvesdulce) [reply src/reducers/api/handlersCMS.js]: The `onRequestLikeComment` doesn't fit in this file. I'd either create the `handlers.js` file again or put it on `api.js`. Please notice that you are exporting but not using the `onRequestLikeComment` anywhere. You should use it here: https://github.com/decred/politeiagui/blob/3bd0e7b8159f8e8064d705e479c472c842b32e69/src/reducers/api/api.js#L167

Comment (amassarwi) [reply src/reducers/api/handlersCMS.js]: @tiagoalvesdulce It's still used in `api.js` - see: https://github.com/amassarwi/politeiagui/blob/localCmsv2/src/reducers/api/api.js#L117

I've re-added the `handler.js` file and moved it.

**Comment by amassarwi** [comment]
> Misplaced icons on the bottom of the proposal detail card

Fixed

**Comment by amassarwi** [comment]
> When 'setting an user as paid', I click on confirm and it only clears the input instead of performing the action

Resolved - wasn't directly related to this diff

**Comment by amassarwi** [comment]
> User-submitted proposals always showing a loading state

Resolved as well.

@fernandoabolafio all three weren't actually related to the merge & reproduced same brhavior on `cmsv2`

**Comment by tiagoalvesdulce** [comment]
Noticed an issue on cms that isn't happening on the cmsv2 branch. The Update button on Manage Contractor isn't being activated when selecting the domain and the contractor type. See:
bugmanage

**Comment by amassarwi** [comment]
@tiagoalvesdulce will investigate further!

**Comment by victorgcramos** [comment]
It's looking good, @amassarwi, good job! I have a couple of adjustments regarding the whole CMS stuff, but I think that it is not this pr's scope. I should open a pr tomorrow fixing that stuff.

LGTM!

**Comment by amassarwi** [comment]
@victorgcramos thanks mate!
I would say let's merge #1831 and then merge this one into master.
After that, we could test one last time on master and open/fix issues if needed.

File: src/apps/cms/config.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Contractor Management",
+  "logoLight": "cms-logo-light.svg",
+  "logoDark": "cms-logo-light.svg",
```
Comment (victorgcramos) [inline src/apps/cms/config.json]: I think this should be replaced by `cms-logo-dark.svg`

File: src/assets/copies/privacy-policy-cms.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+## Privacy Policy
+
+- The [cms.decred.org](cms.decred.org) database stores your account email address, username, cryptographic identity public key(s), IP addresses and invoice transaction details - and associates this data with all of your invoices and comments.
```
Comment (victorgcramos) [inline src/assets/copies/privacy-policy-cms.md]: This URL redirects to `https://github.com/decred/politeiagui/blob/1a8c85c3ee158270cf80e6f2e8214331ecfd76ef/src/assets/copies/cms.decred.org`

File: src/components/DCC/DCCLoader.jsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
+import React from "react";
```
Comment (victorgcramos) [inline src/components/DCC/DCCLoader.jsx]: could you change this file's name to follow the CamelCase pattern we're using? I guess I missed this one when I created it :/

I suggest renaming it to `DccLoader.jsx`.

File: src/apps/cms/config.json (hunk 2 above)
Comment (tiagoalvesdulce) [reply src/apps/cms/config.json]: We cuttently don't have a logo for the dark-theme :/ The one in the assets is a politeia one

**Comment by amassarwi** [comment]
@victorgcramos done mate

**Pull request merged**